react/BarContextProvider/index.spec.jsx

```diff
@@ -64,4 +69,24 @@ describe('BarContextProvider', () => {
     )
     expect(root.html()).toBe('<div>Hello World !<br>6 Jan<br>en</div>')
   })
+
+  it('should try to provide a cozy-intent context', async () => {
+    const client = createMockClient({})
+    const mockStore = configureStore()
+    const store = mockStore(x => x)
+    mount(
```

Comment:
is it wanted to keep using Enzyme instead of React Testing Library?

Reply:
I don't think so, but I didn't want to refactor things here